website/src/pages/use-cases/incident-response.tsx

```diff
@@ -0,0 +1,368 @@
+                                issue.
+                            </p>
+                            <Link
+                                className="font-weight-bold"
+                                to="/case-studies/cloudflare-accelerates-debugging-and-improves-security"
+                            >
+                                Read the Cloudflare case study
+                            </Link>
+                            <img src="/external-logos/cloudflare-color-logo.svg" className="d-block max-w-150 mt-3" />
+                        </div>
+                    </div>
+                </div>
+            </ContentSection>
+        </div>
+
+        <ContentSection>
+            <div className="row mt-lg-0 mt-5 justify-content-center">
+                <div className="d-flex flex-column mt-lg-6 mt-4 w-100 mx-3">
+                    <h1 className="font-weight-bold text-lg-center text-left mb-lg-6 mb-md-6 mb-4">
+                        How Sourcegraph helps
+                    </h1>
+                </div>
+                <div className="pb-lg-5 pb-md-6 pb-5">
+                    <CustomCarousel items={items} autoAdvance={true} smallPanel={true} />
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-gradient-saturn-saturated py-8">
+            <ContentSection>
+                <QuoteCarousel items={quoteCarouselItems} autoAdvance={true} />
+            </ContentSection>
+        </div>
+
+        <div className="bg-light-gray-3 py-7">
+            <ContentSection>
+                <div className="row d-flex flex-column mx-4 mx-lg-0 align-items-lg-center align-items-left">
+                    <div className="mb-5 d-flex flex-column px-lg-7 text-center">
+                        <h1 className="font-weight-bold">Get started with Sourcegraph</h1>
+                        <p>
+                            Respond to incidents with confidence and speed, and remediate issues at their root to ensure
+                            they don't reoccur.
+                        </p>
+                    </div>
+                    <div className="d-flex flex-column">
+                        <Link
+                            className="btn btn-primary"
+                            to="/demo"
+                            title="Request a Demo."
+                            data-button-style={buttonStyle.primary}
+                            data-button-location={buttonLocation.bodyDemo}
+                            data-button-type="cta"
+                        >
+                            Request a demo
+                        </Link>
+                        <Link to="/use-cases" className="d-flex justify-content-center mt-4">
+                            <p className="font-weight-bold">Explore other use cases</p>
+                        </Link>
+                    </div>
+                </div>
+            </ContentSection>
```

Comment:
Same progressive screen adjustment to this section. My commit [here](https://github.com/sourcegraph/about/pull/5271/commits/c523fd6a809b08905bc8bae63e9a21aa0bcd54c0) shows the change I made!
![image](https://user-images.githubusercontent.com/59381432/165857546-65bd58d3-a2e8-4d72-b1b2-65c146035800.png)
TO =>
![image](https://user-images.githubusercontent.com/59381432/165857635-44502f3d-ba12-48bb-b622-05987f530df4.png)

Reply:
Thank you for indicating this, Becca! I agree that the progressive sizing and spacing look much better here.

Reply:
Please see: f76e7e7

Reply:
Awesome! Looks great 🚀